SparkPost/2web2ui
# SA-897 - Health Score V3 visualization.

### What Changed
 - Added the Health Score chart to `signals/health-scoreV3`.

### How To Test
 - Enable the `allow_health_score_V3` flag. 
 - Go to any health score details page. Change the url manually from `signals/health-score` to `signals/health-scoreV3`. Keep the other route params.
 - Check that the graph is working. 

### Notes
Most of the stuff was copied from #1016
